src/main/scala/strawman/collection/mutable/GrowableBuilder.scala

```diff
@@ -0,0 +1,23 @@
+package strawman
+package collection.mutable
+
+import scala.Unit
+
+/** The canonical builder for collections that are growable, i.e. that support an
+  * efficient `+=` method which adds an element to the collection.
+  *
+  * GrowableBuilders can produce only a single instance of the collection they are growing.
+  *
+  * @author Paul Phillips
+  * @version 2.8
+  * @since 2.8
+  *
+  * @define Coll `GrowingBuilder`
+  * @define coll growing builder
+  */
+class GrowableBuilder[Elem, To <: Growable[Elem]](empty: To) extends Builder[Elem, To] {
+  protected var elems: To = empty
+  def clear(): Unit = elems.clear()
+  def result: To = elems
+  def addInPlace(elem: Elem): this.type = { elems += elem; this }
```

Comment:
Having `addInPlace` and `+=` defer to each other seems unnecessarily confusing, especially with a mutable `elems` where `+=` might be changing the identity of `elems`.  (`elems` is a `Growable`, so it's the in place version, but it's still harder to follow.)